roles/openshift_hosted/tasks/router.yml

```diff
@@ -98,28 +86,28 @@
   block:
```

Comment:
This code is duplicated in router and registry.

Consider placing in separate file and passing the data to it.  Might look something like this:
```yaml
include: rollout.yml
vars:
  service: router|registry
when: wait_for_service
```

Reply:
Yeah, I'll do that after we get some basic CI tests passing.